snapshotter/cmd/devmapper/README.md

```diff
@@ -48,6 +48,15 @@ fields:
 * `BaseImageSize` - defines how much space to allocate when creating the base
   device
 
+Here's a sample entry that can be made in the configuration file `/etc/firecracker-dm-snapshotter/config.json`:
+```json
+{
+	"base_image_size" : "20GB",
+	"root_path" : "/tmp/test",
+	"pool_name" : "test"
```

Comment:
I would only add a notice that with this kind of configuration `test` thin-pool need to be prepared in advance.

Reply:
Yes, please add the note.